newIDE/app/src/MainFrame/Preferences/PreferencesDialog.js

```diff
@@ -261,6 +262,21 @@ const PreferencesDialog = ({ i18n, onClose }: Props) => {
               }
             />
           </Line>
+          <Line>
+            <SelectField
+              floatingLabelText={
+                <Trans>Inline popover parameter behavior when escaping</Trans>
```

Comment:
@Wend1go Here the current sentence, your previous ping about the sentence was on an outdated commit.